client/executor/src/wasm_runtime.rs

```diff
@@ -130,6 +131,7 @@ impl RuntimeCache {
 		default_heap_pages: u64,
 		host_functions: &[&'static dyn Function],
 		allow_missing_func_imports: bool,
+		max_runtime_instances: Option<usize>,
```

Comment:
Why isn't that parameter part of the struct + why is this an optional?

IMHO, the default value should be set in the cli.

Reply:
There are many places where executor is constructed without cli. I have moved default value into cli, but I'm not sure that it is better.

Reply:
Constant can still be declared in the executor, but it will result in more dependencies. What do you think?